### ISSUE TITLE
Yoshi code not working

### ISSUE THREAD

**Issue by ZM71** [mkwcat/gecko-codes#9]
Yoshi does not spawn on any stage.

**Comment by mkwcat** [comment]
what code?

**Comment by GH0STP4WZ** [comment]
![floorp_MT07tMo5JZ](https://github.com/user-attachments/assets/c29a0f24-98c6-401b-b103-ed3e14acbb80)
There is no yoshi code. I think OP is a bit confused.

**Comment by ZM71** [comment]
Then there's on another site

**Comment by GH0STP4WZ** [comment]
If you mean codes on another site, then this is the wrong repo to report issues to, this repository is only for @mkwcat's codes. Not any other code.

**Comment by ZM71** [comment]
I though this was a collection of all codes for NSMBW Wii?

**Comment by GH0STP4WZ** [comment]
Yes, but only the codes MKW cat have made are in this repo. And this issues page is only for any of the codes in this repo. The code you speak of (a code relating to yoshi) is not in this repository, therefore we cannot help you.

**Comment by ZM71** [comment]
I confused this with something else sorry
